docs/topics/modules/creating_new_project.adoc

```diff
@@ -3,20 +3,21 @@
 
 // for hello-world
 ifeval::["{context}" == "hello-world"]
-When you create a new Space, the `How would you like to get started?` menu appears. Use this menu to create a new Quickstart project as follows:
+When you create a new Space, the `How would you like to get started?` menu opens. Use this menu to create a new Quickstart project as follows:
```

Comment:
Same as above. Happy to discuss and go with whatever the majority prefers but using opens for menus or applications feels wrong. I'd recommend displays if it is just appearing in the page.

Reply:
In the case of menus, I think 'appears, opens, is displayed' are all good.

Reply:
Well this is the reference from IBM which started this thread; let me know what you folks think: >appear v
Do not use unless there is no better alternative. Possible alternatives include “is displayed” and “opens.”
“Appear” implies magic or illusion (for example, “it appears to be snowing, but it is the falling volcano
ash”). When you are referring to a window in a graphical user interface, always use “open.” Examples: (1)
“The message is displayed when you make a mistake.” (2) “The window opens...”

Reply:
Right, let's use 'open' for windows (incl. dialog boxes) and 'display' for menus and other UI elements.

That said, given the state of things in OSIO, 'appears' seems to be just about right. I think there's a fair amount of magic involved these days...

_"Turn around counter-clockwise three times, spit over your left shoulder, mutter the incantation, and if Jupiter is well aligned with Venus, the menu will appear. Make your selection as fast as possible before the menu disappears in a puff of smoke."_